### PR TITLE
Remove panics from code 

### PR DESCRIPTION
**Pull request by grokkos**
Replaced panics with debugging messages and err's

- Todo check the places we need to return the error instead of just info debugging messages and the refactor effort needed in the whole codebase
- Test how the removal of the panics will affect the efficiency and consistency, in extreme occasions that is possibly needed the app to throw panics in order to prevent misbehaviour.

### CODE REVIEW

File: ios/accessibility/accessibility_control.go [hunk 1]
```diff
@@ -19,10 +17,10 @@ func (a ControlInterface) readhostAppStateChanged() {
 		msg := a.channel.ReceiveMethodCall("hostAppStateChanged:")
 		stateChange, err := nskeyedarchiver.Unarchive(msg.Auxiliary.GetArguments()[0].([]byte))
 		if err != nil {
-			panic(err)
+			logrus.WithError(err).Error("Failed unarchiving NSKeyedArchiver plists")
```
Comment (danielpaulus) [inline ios/accessibility/accessibility_control.go:20]: Can you please change back `logrus` to log? 
this is a little verbose for me :-)

**Review by danielpaulus — CHANGES_REQUESTED** [review]

File: ios/accessibility/accessibility_control.go (hunk 1 above)
Comment (grokkos) [reply ios/accessibility/accessibility_control.go]: sure! done it in all the other places as well, let me know if it looks good or I missed smt :D

File: restapi/api/util.go [hunk 2]
```diff
@@ -31,15 +31,15 @@ func GetVersion() string {
 func MustMarshal(v interface{}) string {
 	b, err := json.Marshal(v)
 	if err != nil {
-		panic(err)
+		log.WithError(err).Error("Failed marshalling object")
```
Comment (danielpaulus) [inline restapi/api/util.go:34]: If it's in a `MustXXX` function then throwing a panic is ok. Also the restapi is not library code, so panic'ing here is usually ok.

File: restapi/api/util.go [hunk 3]
```diff
@@ -10,16 +10,16 @@ import (
...
 )
 
 type GenericResponse struct {
 	Message string `json:"message,omitempty"`
 	Error   string `json:"error,omitempty"`
 }
 
-//GetVersion reads the contents of the file version.txt and returns it.
-//If the file cannot be read, it returns "could not read version"
+// GetVersion reads the contents of the file version.txt and returns it.
```
Comment (danielpaulus) [inline restapi/api/util.go:21]: While I don't have a problem with adding these whitespaces in general, it usually is a good practice not to mix code cleanup/automated refactoring with your changes. It makes it hard to review and is a risk of adding merge conflicts. Please remove them from this PR, I am ok with merging them separately.

**Review by danielpaulus — CHANGES_REQUESTED** [review]

**Pull request closed without merging**